rollup.config.js

```diff
@@ -0,0 +1,15 @@
+const { readdirSync } = require("fs");
+
+/**
+ * Rollup configuration to build correctly our scripts (nodejs scripts need to be cjs)
+ */
+module.exports = readdirSync("lib/bin")
```

Comment:
Is rollup really needed for this? TypeScript supports a commonjs export...

Reply:
Yes, because the tsconfig is managed by operational-scripts